src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int base1;
```

Comment:
Better don't use numbers in variables names. Let's rename it to `baseOne` or `firstBase`

Reply:
Done